### PR TITLE
Add aimec analysis results to report

### PR DESCRIPTION
**Pull request by awirb**
* add table to report 
* adjust ini files for aimec of benchmarks

### CODE REVIEW

File: avaframe/log2Report/generateCompareReport.py [hunk 1]
```diff
@@ -133,6 +133,31 @@ def writeCompareReport(reportFile, reportD, benchD, avaName, cfgRep):
...
+        for value in parameterList:
+            # if reference and simulation parameter value do not match
+            # mark red
+            if valuesBench[countValue] != valuesSim[countValue]:
+                valueRed = '<span style="color:red"> %.4f </span>' % valuesSim[countValue]
+                pfile.write('| %s | %.4f | %s | \n' % (value, valuesBench[countValue], valueRed))
+                countValue = countValue + 1
+            else:
+                pfile.write('| %s | %.4f | %.4f | \n' % (value, valuesBench[countValue], valuesSim[countValue]))
+                countValue = countValue + 1
```
Comment (fso42) [inline avaframe/log2Report/generateCompareReport.py:155]: I would drop most, if not all decimals. sindce they are all treated the same, I would use .4f...

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 2]
```diff
@@ -327,6 +327,7 @@ def postProcessAIMEC(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
     runoutLength = runout[0]
     TP, FN, FP, TN, _ = aT.analyzeArea(rasterTransfo, runoutLength, pLim, dataPressure, cfgPath, cfgFlags)
 
+    print('RUNOUT', runout)
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:330]: debug print? or remove?

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 3]
```diff
@@ -426,3 +427,25 @@ def postProcessAIMECReport(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
     resAnalysis['areasPlot'] = {'Aimec area analysis': compPlotPath}
 
     return resAnalysis
+
+
+
+def aimecRes2ReportD(resAnalysis, reportD, benchD, refSim):
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:433]:
```suggestion
def aimecRes2ReportDict(resAnalysis, reportD, benchD, refSim):
```
?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/log2Report/generateCompareReport.py (hunk 1 above)
Comment (awirb) [reply avaframe/log2Report/generateCompareReport.py]: I changed  it to %.2f

**Review by fso42 — APPROVED** [review]

**Pull request merged**